docs/criteria/catalogs/rc.md

```diff
@@ -0,0 +1,51 @@
+# Spice.ai OSS Catalog Connectors - RC Criteria
+
+This document defines the set of criteria that is required before a Catalog Connector is considered to be of [RC](../definitions.md) quality.
+
+All criteria must be met for the Catalog to be considered [RC](../definitions.md), with exceptions only permitted in some circumstances (e.g. it would be technically infeasible to add a feature/fix a bug for a particular Catalog).
+
+## RC Quality Catalogs
+
+| Catalog       | RC Quality | DRI Sign-off |
+| ------------- | ------------ | ------------ |
+| Databricks    | ➖           |              |
+| Iceberg       | ➖           |              |
+| Spice.ai      | ➖           |              |
+| Unity Catalog | ➖           |              |
+
+## RC Criteria
+
+The RC criteria expand on and require that all [Beta release criteria](./beta.md) continue to pass for the Catalog.
+
+- [ ] All [Beta release criteria](./beta.md) pass for this Catalog.
+
+Catalogs that use existing Data Connectors requires that the Data Connector must first be at RC.
+
+Catalogs without an existing Data Connector must perform the RC Data Connector criteria in addition to the Catalog criteria.
+
+### Existing Data Connector
+
+This section is required if the Catalog uses an existing Data Connector.
+
+- [ ] The associated Data Connector for this Catalog is currently of RC quality or higher.
+
+### No Data Connector
+
+This section is required if the Catalog does not use an existing Data Connector.
+
+- [ ] The Catalog passes the [RC Data Connector release criteria](../connectors/rc.md)
+
+### All Catalogs
+
+- [ ] The Catalog can refresh the list of datasets, and add/remove datasets that are available.
+  - [ ] Catalog refresh is made available via an API endpoint to trigger on-demand Catalog refreshes.
```

Comment:
```suggestion
```

This doesn't need to be at an individual catalog connector level - it should just be base functionality provided by the runtime that uses the the catalog connector.